self-hosting/GUIDE.md

```diff
@@ -0,0 +1,216 @@
+# Self Hosting
+
+- [Docker Compose](#docker-compose)
+- [Nginx Reverse Proxy](#nginx-reverse-proxy)
+- [Cloudflare Tunnel](#cloudflare-tunnel)
+
+## Docker Compose
+
+We recommend using Docker-compose for the simplest way to deploy Omnivore. We have provided a configuration in the `self-hosting/docker-compose` folder. 
+
+All networking and persistent storage is handled by the docker-compose file.
+
+### Requirements
+* Docker 
+* Docker Compose 
+
+### 1. Clone the Repository
+
+Clone the repository at ``git@github.com:omnivore-app/omnivore.git``
+
+### 2. Change directory to self-hosting/docker-compose
+
+The Docker-compose file and necessary environment variables are found in the self-hosting folder under docker-compose. 
+
+These files provide all you need to get Omnivore up and running on your local environment. 
+
+### 3. Populate the .env file 
+
+There is a .env.example file located within the docker-compose folder that should give you the necessary environment variables to begin running. 
+You can use these by `mv .env.example.env`
+
+The following environment variables should be changed to reflect where you are running your application. 
```

Comment:
I believe the table below should also include `IMAGE_PROXY_URL` and `CLIENT_URL` as variables that need to be changed.